.travis.yml

```diff
@@ -60,5 +60,8 @@ script:
   - yarn test:setup
   - yarn test
 
+after_success:
+  - yarn run publish
```

Comment:
These haven't been invoked yet because we haven't prioritized the server-side work to consume these releases and co-ordinate our deployments.